contrib/plugins/plugin-clair/clair.go

```diff
@@ -69,12 +69,41 @@ func (d ClairPlugin) Run(a plugin.IJob) plugin.Result {
 	_ = plugin.SendLog(a, "Running analysis")
 	analysis := clair.Analyze(image, manifest)
 
-	json, err := jsoniter.Marshal(analysis)
-	if err != nil {
-		_ = plugin.SendLog(a, "Unable to push image on Clair: %s", err)
+	_ = plugin.SendLog(a, "Creating report")
+
+	var vulnerabilities []sdk.Vulnerability
+	summary := make(map[string]int64, 0)
```

Comment:
should use make(map[string]int64) instead

Reply:
fixed